docs/contributors.json

```diff
@@ -1,32 +1,122 @@
 [
-  "antfu",
-  "sheremet-va",
-  "patak-dev",
-  "Aslemammad",
-  "userquin",
-  "DerYeger",
-  "Demivan",
-  "edimitchel",
-  "Saul-Mirone",
-  "11joselu",
-  "dammy001",
-  "tony-go",
-  "wtchnm",
-  "cexbrayat",
-  "dominikg",
-  "mitchelvanbever",
-  "tigerabrodi",
-  "horacioh",
-  "LeonardSSH",
-  "sachinraja",
-  "cawa-93",
-  "Monkatraz",
-  "sudongyuer",
-  "hannoeru",
-  "JakeGinnivan",
-  "JoostK",
-  "PuruVJ",
-  "aleclarson",
-  "CyriacBr",
-  "christianhg"
-]
\ No newline at end of file
+  [
+    "antfu",
+    "https://avatars.githubusercontent.com/u/11247099?v=4&s="
```

Comment:
Can't we just use github.com/<name>.png?

Reply:
yes, but there will be N + M client redirections (where N is the number of core team members and M is the number of contributors)